common/build/build-common/.changeset/rich-candles-tan.md

```diff
@@ -0,0 +1,12 @@
+---
+"@fluidframework/build-common": minor
+---
+
+Strict API-Extractor config now fails on incompatible release tags
```

Comment:
@tylerbutler Are there any noteable restrictions to the kinds of markdown contents we can include in these? E.g. could I link to the API-Extractor docs for the "incompatible release tags" rule here?

Reply:
Yeah, you can include any markdown formatting.

Reply:
Though I think the first line is considered a summary so probably avoid formatting there. Regardless, we can change this safely any time before release.